### ISSUE TITLE
Please add more demos

### ISSUE THREAD

**Issue by cubiclesoft** [blueimp/jQuery-File-Upload#3101]
In my experience, a project of this size and scope generally has at least 10 different demos.  However, this project only has 3 demos ("Basic Plus UI", "Angular JS", and "jQuery UI" demos all basically do the same thing, therefore they are essentially one demo).

Ideas for demos:

Each FAQ item on the "Basic plugin" and other documentation pages, for example, could each be their own individual demo.  Doing just this would easily add about 8 demos.

A demo of a sequential uploader that shows two progress bars.  The first bar shows the total upload progress (number of files in the queue, total bytes transferred, and total bytes), the second bar shows current file upload progress (bytes sent, total bytes, time left, bitrate).  A lot of server-side code only supports handling one file at a time, so a demo like this would satisfy many developers.

A demo showing various callback functions and the "useful" values of the 'data' object in each callback using a textarea (or similar) to store the log.  This would make it easier to understand what callbacks are called in what order and what data the callbacks have available.

A demo using chunked uploads.

A demo showing how to allow multiple files to be dragged-and-dropped, but only uploading those in the set with specific file extensions.

A demo showing HTML 5 data attribute support.

A demo showing the 'enable', 'disable', and 'destroy' API options.

A demo showing programmatic upload options.

A demo showing how to delay-load the plugin.

In summary, more demos = better.  The more demos there are, the fewer questions people will have regarding usage of the plugin.  In addition, the documentation can be simplified to the point that the bulk of it won't be needed any more.

**Comment by blueimp** [comment]
Looking back at this after more than 5 years, I do think that all requests for better documentation are valid.
So apologies to @cubiclesoft, @camdagr8, @Undergarun and @julianwap for not providing better documentation and for suggesting to add it yourself, which was unwarranted. 

Unfortunately, this project also posed a great maintenance burden over the years and I've actually removed the alternate demo versions in this commit: 3142b42
The last tagged version that included them was v9.34.0
